test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -64,7 +64,7 @@ Describe "Measure-Object" -Tags "CI" {
     }
 
     It "Should be able to count using the Property switch" {
-        $expected = $(Get-ChildItem $TestDrive).Length
+        $expected = $(Get-ChildItem $TestDrive | Where-Object { $_ -is [System.IO.FileInfo] }).Length
```

Comment:
Isn't this test relying on there being something in $TestDrive?  If it happens to be empty and Measure-Object incorrectly returns 0, the test would pass.  Seems like we should make this test more predictable by creating a known set of files/directories in there and probably a specific subfolder under $testdrive.

Reply:
Alternatively since this test is for Measure-Object and not the filesystem, perhaps count some other type of object.

Reply:
Should also have a test case to cover the error case for Property

Reply:
I've made the test independent of the filesystem.

Since this PR was just to fix the CI failure, the extra test is out of scope I think.